### ISSUE TITLE
ImageNet training gets "Killed" response

### ISSUE THREAD

**Issue by GrayGriffin** [xiangning-chen/SmoothDARTS#9]
I have been attempting to run the imagenet_train program on Linux, and when I try I only get to this stage before it displays "Killed." The computer I am using has 31.1 GB of memory, which is already more than the previous computer I tried this on, so I'm not sure how much I actually need.
<img width="284" alt="Screen Shot 0111-08-24 at 2 10 39 PM" src="https://user-images.githubusercontent.com/13948771/186343837-d233f0c6-5dd1-44fe-a48d-1ddc71fec0e8.png">

**Comment by xiangning-chen** [comment]
Hi,

Thanks for your interest.
I used 8 1080ti for training on ImageNet, probably you can try to reduce the batch size and tune the learning rate accordingly.

best
